### PR TITLE
README

### PR DESCRIPTION
**Pull request by hokiegeek2**
This PR moves the README.md file from the client directory to the project root and adds more content

### CODE REVIEW

File: arkouda_metrics_exporter/README.md [hunk 1]
```diff
@@ -2,16 +2,16 @@
...
+2. Updates corresponding Prometheus data structures cached in the ArkoudaMetrics object
 3. Provides a Prometheus scrape target
 
 The arkouda_metrics_exporter project leverages the Prometheus [prometheus-client](https://github.com/prometheus/client_python) for all Prometheus metrics as well as the embedded http_server to deliver the Prometheus scrape target.
 
 
 ## Installation
 
-arkouda_metrics_exporter depends upon [arkouda](https://github.com/Bears-R-Us/arkouda) and [prometheus_client](https://pypi.org/project/prometheus-client/). Since there currently is no pypi Arkouda installer, an arkouda clone/code download followed by pip install precedes installing arkouda_metrics_exporter:
+arkouda_metrics_exporter depends upon [Arkouda](https://github.com/Bears-R-Us/arkouda) and [prometheus_client](https://pypi.org/project/prometheus-client/). Since there currently is no pypi Arkouda installer, an arkouda clone/code download followed by pip install precedes installing arkouda_metrics_exporter:
```
Comment (Ethan-DeBandi99) [inline arkouda_metrics_exporter/README.md:14]: Should we point to the `INSTALL.md` in the `arkouda` repo for installation instructions?

**Review by Ethan-DeBandi99 — COMMENTED** [review]
One question. Not sure if we need to do what I am asking or not so left as a comment only.

File: arkouda_metrics_exporter/README.md (hunk 1 above)
Comment (hokiegeek2) [reply arkouda_metrics_exporter/README.md:14]: @Ethan-DeBandi99 good point, I will update now

Comment (hokiegeek2) [reply arkouda_metrics_exporter/README.md]: @Ethan-DeBandi99 done!

**Review by Ethan-DeBandi99 — APPROVED** [review]

**Pull request merged**